src/Microsoft.Diagnostics.Monitoring.WebApi/Exceptions/ExceptionsConfiguration.cs

```diff
@@ -0,0 +1,134 @@
+            {
+                if (evaluateFilterList(configuration, topFrame))
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        internal bool ShouldInclude(IExceptionInstance exception)
+        {
+            Func<ExceptionConfiguration, CallStackFrame, bool> evaluateFilterList = (configuration, topFrame) =>
+            {
+                bool include = true;
+                if (topFrame != null)
+                {
+                    CompareIncludeValues(configuration.MethodName, topFrame.MethodName, ref include);
+                    CompareIncludeValues(configuration.ModuleName, topFrame.ModuleName, ref include);
+                    CompareIncludeValues(configuration.ClassName, topFrame.ClassName, ref include);
+                }
+
+                CompareIncludeValues(configuration.ExceptionType, exception.TypeName, ref include);
+
+                return include;
+            };
+
+            return CheckConfiguration(exception, Include, evaluateFilterList);
+        }
+
+        internal bool ShouldExclude(IExceptionInstance exception)
+        {
+            Func<ExceptionConfiguration, CallStackFrame, bool> evaluateFilterList = (configuration, topFrame) =>
+            {
+                bool? exclude = null;
+                if (topFrame != null)
+                {
+                    CompareExcludeValues(configuration.MethodName, topFrame.MethodName, ref exclude);
+                    CompareExcludeValues(configuration.ModuleName, topFrame.ModuleName, ref exclude);
+                    CompareExcludeValues(configuration.ClassName, topFrame.ClassName, ref exclude);
+                    CompareExcludeValues(configuration.ExceptionType, exception.TypeName, ref exclude);
+                }
+                else
+                {
+                    CompareExcludeValues(configuration.ExceptionType, exception.TypeName, ref exclude);
+                }
+
+                return exclude ?? false;
+            };
+
+            return CheckConfiguration(exception, Exclude, evaluateFilterList);
+        }
+
+        private static void CompareIncludeValues(string configurationValue, string actualValue, ref bool include)
+        {
+            if (include && !string.IsNullOrEmpty(configurationValue))
+            {
+                include = CompareValues(configurationValue, actualValue);
+            }
+        }
+
+        private static void CompareExcludeValues(string configurationValue, string actualValue, ref bool? exclude)
+        {
+            if (exclude != false && !string.IsNullOrEmpty(configurationValue))
+            {
+                exclude = CompareValues(configurationValue, actualValue);
+            }
+        }
+
+        private static bool CompareValues(string configurationValue, string actualValue)
+        {
+            return actualValue.Contains(configurationValue, StringComparison.CurrentCultureIgnoreCase);
```

Comment:
I covered a few of these things in https://github.com/dotnet/dotnet-monitor/pull/5131#discussion_r1291765141 - but in summary I'd rather bias this toward more results than less. In those examples, a user starts with something simple - if they get more results than they wanted, they can further refine their query. One case where this does fall apart (which may/may not be a big issue, and I can change this if we feel that it is) is users who have methods called "MyMethod", "MyMethod2", "MyMethod3", etc. - any search for "MyMethod" inherently includes "MyMethod2/3/etc.", unless you explicitly exclude "MyMethod2/3/etc." (which is a bit tedious).

In this case I'm not sure if the standardization makes sense because it's just for easier consumption, not because we actually need to know exactly what method the user means. For something like this, having to remember to type "MyNameSpace.MyClass+MyNestedType" instead of "MyClass" feels excessive - it's less precise, but that feels OK to me for this use-case...I think this will still do a good job of using broad strokes to eliminate most of what you don't care about